src/test/java/client/PetClient.java

```diff
@@ -40,6 +44,15 @@ public static RequestSpecification buildReq() throws IOException {
             .build();
     }
 
+    @Step("Building request specification")
+    public static RequestSpecification buildUncheckedReq() throws IOException {
+        LOGGER.debug("building request specification ");
+
+        return new RequestSpecBuilder()
+            .setBaseUri(getBaseUrl("PET_STORE_HOST"))
```

Comment:
Do you ever use any other hosts? Consider removing input parameter for `getBaseUrl()` method altogether

Reply:
should I put this method it in petClient class?